Dockerfile

```diff
@@ -29,6 +29,9 @@ RUN echo -e "keyserver-options auto-key-retrieve" >> /etc/pacman.d/gnupg/gpg.con
 RUN echo -e "#!/bin/bash\nif [[ \"$1\" == \"--version\" ]]; then echo 'fake 244 version'; fi\nmkdir -p /var/cache/pikaur\n" >> /usr/bin/systemd-run && \
     chmod +x /usr/bin/systemd-run
 
+# substitute check with !check to avoid running software from AUR in the build machine
+RUN sed -i -e 's/BUILDENV=(!distcc color !ccache check !sign)/BUILDENV=(!distcc color !ccache !check !sign)/g' /etc/makepkg.conf
```

Comment:
"Check" is found in multiple occurrences. I think they all are inside comments, but will they ever be? The problem of change will present itself, whatever option is chosen. However, in case arch changes that line chimera will compile itself (just slower) if however I replace just "check" that has way higher probability to cause a big breakage.

Reply:
It is fine. You can do a fancy line that will look for "BUILDENV=" at the begginning of the line and then repalce only the "check". But will suffice for now.
That line don't change that often upstream and we can revisit it later.